src/components/app.tsx

```diff
@@ -167,4 +137,4 @@ class AppBase extends React.PureComponent<Props, {}> {
   }
 }
 
-export const App = DragDropContext(HTML5Backend)(AppBase);
+export const App = AppBase;
```

Comment:
There is no wrapping anymore.  Get rid of this line and just rename AppBase to App?